README.md

```diff
@@ -79,6 +79,9 @@ const MyAppleSigninButton = () => (
 export default MyAppleSigninButton;
 ```
 
+##### :warning: Disclaimer
+``onSuccess`` response object will contain the user object on first time attempt. Meaning if you make another signin attempt you will not get the user object. This is true for third and N-th time attempt of the same account.
+
```

Comment:
Is the user object omitted on the second or third attempt?

Reply:
for any signin attempt after the first.